pyqgisserver/plugins.py

```diff
@@ -54,37 +54,43 @@ def find_plugins(path: str) -> Generator[str,None,None]:
     """
     path = Path(path)
     for plugin in path.glob("*"):
-        LOGGER.debug("Looking for plugin in %s", plugin)
+        LOGGER.debug(f"Looking for plugin in {plugin}")
+
+        # Plugin folder can be a symlink
+        plugin.resolve()
+
         if not plugin.is_dir():
+            LOGGER.warning(f"'{plugin}' is not a directory. Discarding")
```

Comment:
Useless warning wich will bloat output: the search  for plugin is recursive  is this will output warning for every file or dir.